app/javascript/gobierto_visualizations/webapp/lib/config/category_titles.js

```diff
@@ -0,0 +1,74 @@
+export const categoryTitleEng = [
```

Comment:
these are not exactly the titles in English, these are the translation keys. I'm saying it because maybe in the future there are real English translations and they object should have this name (just like `categoryTitlesCat` and `categoryTitlesEsp`).